src/components/frame/LandingPage/LandingPageItem.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { FormattedMessage, MessageDescriptor } from 'react-intl';
+import clsx from 'clsx';
+
+import Icon from '~shared/Icon';
+import Heading from '~shared/Heading';
+import Button from '~v5/shared/Button';
+
+const displayName = 'frame.LandingPage';
+
+interface Props {
+  iconName: string;
+  headingText: MessageDescriptor;
+  headingDescription: MessageDescriptor;
+  buttonText: MessageDescriptor;
+  onClick: () => void;
+  itemIndex: number;
+  onHover: (itemIndex: number) => void;
+  disabled?: boolean;
+}
+
+const LandingPageItem = ({
+  iconName,
+  headingText,
+  headingDescription,
+  buttonText,
+  onClick,
+  disabled,
+  onHover,
+  itemIndex,
+}: Props) => {
+  return (
+    <div
+      className={clsx(
+        'group flex items-center justify-between px-6 py-5 rounded-lg border border-gray-200',
+        { 'hover:border-blue-400': !disabled },
+      )}
+      onMouseEnter={() => onHover(disabled ? 0 : itemIndex)}
+      onMouseLeave={() => onHover(0)}
+    >
+      <div className="flex items-center">
+        <div
+          className={clsx(
+            'flex items-center px-[23px] py-[25px] mr-4 bg-base-bg rounded-lg',
```

Comment:
Is this one pixel here really important? What about w just use `p-6` (24px)? Would that be terrible, @melyndav?
I usually just round to the nearest tailwind value. Makes things more consistent. There's a reason they made it this way.

Reply:
@chmanie 24px is fine. Great pick up. :)